whereby/sdk
# browser-sdk: Replace storybook webpack with vite

### Description

Storybook HMR is currently broken. This fixes that, but also changes the storybook runner to `vite` instead of `webpack`. This is faster and in line with the example apps in this repo (all of them use `vite`). This allowed me to remove all `babel` dependencies in browser-sdk, as they are only used for storybook. 

**Summary:**



**Related Issue:**



### Testing

1. `yarn install`
2. `yarn build`
3. `yarn dev`
4. Storybook should start
5. Change a file in the browser-sdk package (add a console log or something)
6. Verify that the storybook hot module reloading works and the page is refreshed and updated with the changes.

The log here says "Could not fast refresh" but that's OK - we didn't have that before either. It will do a full page reload when you do changes in the code instead.
![image](https://github.com/whereby/sdk/assets/6151521/cb8e06f2-a58d-4272-b499-bcb8e39716c6)



### Screenshots/GIFs (if applicable)



### Checklist

        under the terms of the MIT license.

### Additional Information
